open-apparel-registry/open-apparel-registry
# Store CSV header row on the FacilityList model

NOTE that this is built on top of #65. Wait until #65 is merged, then retarget this to `develop`.

## Overview

Allows us to reconstruct the full uploaded file and guides the parsing of raw file lines into country, name, and address fields.

Connects #72

## Testing Instructions

* Reset your database by running `./scripts/manage flush` and `./scripts/manage migrate api zero`
* Verify that `./scripts/manage migrate` and `./scripts/manage loadfixtures` run without error.
